test/distributed/test_c10d_gloo.py

```diff
@@ -640,6 +640,7 @@ def _test_sparse_allreduce_basics(self, fn):
                 self.assertEqual(tensors, outputs)
                 self.assertEqual(result, outputs)
 
+    @unittest.skip("intermittent failures on Windows, in CI")
```

Comment:
Don't we want to disable it just for Windows?

Reply:
shooot, I'll do another one!

Reply:
Can we also add some context here pointing to a gh issue to follow up on?